src/workflow/mod.rs

```diff
@@ -175,3 +175,35 @@ impl WorkflowManager {
         Ok(())
     }
 }
+
+/// Determines when workflows are kept in the cache or evicted
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum WorkflowCachingPolicy {
```

Comment:
Seems like this would be handy for debugging purposes but we probably don't want to expose any of this to the user other than `max_cached_workflows`.

Reply:
Yeah, Max mentioned we don't want to expose the non sticky mode. So, I'll have this not be an enum. To be clear, `AfterEveryReply` would never show up publicly because it's compile-time guarded to be test only. It makes this slightly annoying b/c having this be an enum is nice ergonomically inside core, but it ends up only having one variant publicly, but oh well.